### ISSUE TITLE
Oncotree mappings at the tissue level

### ISSUE THREAD

**Issue by bgyori** [monarch-initiative/mondo#8008]
I have been looking at the Oncotree resource and noticed that MONDO provides xrefs to it. Oncotree is organized in a way that at the top level, it lists tissues (e.g., LUNG, PANCREAS, etc.) and all descendent terms are types of cancer (e.g., LUAD, PAAD). This tissue-level entries create a bit of a confusing situation in terms of mappings to MONDO, for instance (line numbers included):

```
468005 [Term]                                                                          
468006 id: MONDO:0021117                                                               
468007 name: lung neoplasm                                                             
468008 def: "A benign or malignant, primary or metastatic neoplasm involving the lungs. Representative examples of benign                 neoplasms include adenoma, papilloma, chondroma, and endobronchial lipoma. Representative examples of malignant neoplasms          include carcinoma, carcinoid tumor, sarcoma, and lymphoma." [NCIT:C3200]       
468009 subset: otar {source="MONDO:OTAR"}                                              
468010 synonym: "lung" RELATED [ONCOTREE:LUNG]                                         
468011 synonym: "lung neoplasm" EXACT [NCIT:C3200]                                     
...                          
468026 xref: MEDGEN:7400 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}          
468027 xref: MESH:D008175 {source="MONDO:equivalentTo"}                                
468028 xref: NCIT:C12468 {source="ONCOTREE:LUNG"}**                                      
468029 xref: NCIT:C3200 {source="MONDO:equivalentTo"}                                  
468030 xref: ONCOTREE:LUNG {source="MONDO:equivalentTo"}                               
468031 xref: UMLS:C0024121 {source="MEDGEN:7400", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
468032 is_a: MONDO:0005070 {source="MESH:D008175/inferred", source="MONDO:Redundant", source="NCIT:C3200/inferred"} ! neoplasm
468033 is_a: MONDO:0005275 {source="MESH:D008175", source="MONDO:Redundant", source="NCIT:C3200"} ! lung disorder
468034 is_a: MONDO:0021350 {source="MONDO:Redundant", source="NCIT:C3200"} ! neoplasm of thorax
468035 intersection_of: MONDO:0005070 ! neoplasm                                       
468036 intersection_of: disease_has_location UBERON:0002048 ! lung                     
```

Here, the xref
```
ONCOTREE:LUNG {source="MONDO:equivalentTo"}
```
would imply an exact equivalence. But the fact that ONCOTREE:LUNG is a tissue, not a cancer type results in a conflict in terms of NCIT mappings:
```
xref: NCIT:C12468 {source="ONCOTREE:LUNG"}                                      
xref: NCIT:C3200 {source="MONDO:equivalentTo"}           
```
where NCIT:C12468 is just "lung" and NCIT:C3200 is "lung neoplasm".

My question is, is this way of representing ONCOTREE tissue-level entries intentional or should e.g. ONCOTREE:LUNG {source="MONDO:equivalentTo"} be changed to a different type of relationship? If it is intentional, I will figure out how to resolve the xref issues, just want to make sure!

**Comment by cmungall** [comment]
TL;DR - it looks like you are using the mondo-edit file. In all release artefacts this should not be there

`xref: NCIT:C12468 {source="ONCOTREE:LUNG"}    `

but either way we should make the edit file less confusing and clarify if and how MONDO:0021117   should be mapped to oncotree

AFAICR the oncotree tissue terms could be interpreted in 2 ways

1. as tissues with equivalents in UBERON
2. as neoplasm terms that are labeled as tissue terms 

The paper leaves in IMO open:

"its root node represents a tissue and the branches and leaves include different histological and genomic subclassifications"

However, if Oncotree is already mapped to NCIT tissues then this points to 1.

1 is a little less practical as it divides the mapping task across two ontologies, OR forces us to use an odd way of representing the non-equivalent cognate term.

If we want to stick with 2, we want to eliminate the erroneous propagated xrefs to NCIT (they are already blocked from the release but their presence in the edit files does no good and could confuse us internally)

**Comment by bgyori** [comment]
Hi @cmungall! Thanks, that all makes sense.
- Yes, I've been looking at mondo-edit.obo, and didn't realize that the release would be different in this regard, that's good to know.
- I agree that an interpretation of Oncotree where top level terms represent "cancer originating in [TISSUE]" would make sense, but given their mappings to NCIT tissue terms, it seems like the intended interpretation is "[TISSUE] of origin".
